nodeup/pkg/model/kube_apiserver.go

```diff
@@ -318,6 +352,42 @@ func (b *KubeAPIServerBuilder) buildPod() (*v1.Pod, error) {
 
 	pod.Spec.Containers = append(pod.Spec.Containers, *container)
 
+	if b.Cluster.Spec.EncryptionConfig != nil && *b.Cluster.Spec.EncryptionConfig {
+		sideCar := &v1.Container{
+			Name:  "kms-plugin",
+			Image: "sethpollack/kmsplugin:latest",
+			Args: []string{
+				"-listen",
+				"/var/run/kmsplugin/socket.sock",
+				"-key",
+				keyName,
+			},
+			LivenessProbe: &v1.Probe{
+				Handler: v1.Handler{
+					HTTPGet: probeAction,
+				},
+				InitialDelaySeconds: 15,
+				TimeoutSeconds:      15,
+			},
+			Ports: []v1.ContainerPort{
+				{
+					Name:          "local",
+					ContainerPort: 8080,
+					HostPort:      8080,
+				},
+			},
+			Resources: v1.ResourceRequirements{
+				Requests: v1.ResourceList{
+					v1.ResourceCPU: resource.MustParse("150m"),
```

Comment:
This is likely to oversubscribe the master on single-core machines.  Maybe set it to 10m for now.

We (manually) track the usage here: https://github.com/kubernetes/kops/blob/98b2437bc001907da8d4632b8476d7865f797eb1/docs/development/instancesizes.md